packages/taquito/src/operations/types.ts

```diff
@@ -82,7 +84,7 @@ export type RPCOpWithSource =
 
 export const isOpWithFee = <T extends { kind: OpKind }>(
   op: T
-): op is withKind<T, Exclude<OpKind, OpKind.ACTIVATION>> => {
+): op is withKind<T, Exclude<Exclude<OpKind, OpKind.ACTIVATION>, OpKind.FAILING_NOOP>> => {
```

Comment:
unsure if it's needed to exclude failing_noop

Reply:
I think it does. We will have a build time error without that